docs/docs/api/http.md

```diff
@@ -663,3 +662,16 @@ information.
 ```
 
 This endpoint returns `200` if the operation was successful and `400` otherwise.
+
+### Removing metadata
+
+`POST /metadata.remove`
+
+```json
+{
+  "conversation_id": "conversation-id",
+  "key": "ad.id"
+}
+```
+
+This endpoint returns `200` if the operation was successful and `400` otherwise.
```

Comment:
I don't think this can return 400, right? 🤔

Reply:
Typo, will change it